src/components/views/createplayer/LandingPage.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react"
+import { usePlayers } from "../../../hooks/api/usePlayers"
+import { useNavigate } from "react-router-dom"
+import { Paths } from "../../routing/routers/Paths"
+import Title from "../../../img/titleImage.png"
+import Oberin from "../../../img/oberin.png"
+
+import "../../../styles/fonts.scss"
+import "../../../styles/ui/Box.scss"
+import "../../../styles/ui/Divs.scss"
+import "../../../styles/ui/images.scss"
+import "./LandingPage.scss"
+
+const LandingPage = () => {
+  const navigate = useNavigate()
+  const [name, setName] = useState("")
+  const changeName = (e) => {
+    setName(e.target.value)
+  }
+
+  const { me, loading, createPlayer, startPollPlayers } = usePlayers()
+
+  const submit = async (e) => {
+    e.preventDefault()
+    await createPlayer(name)
+    navigate(Paths.LOBBY)
+  }
+
+  useEffect(() => {
+    if (me !== null) {
+      navigate(Paths.LOBBY)
+    }
+    const playersSubscription = startPollPlayers()
+    return () => playersSubscription.cancel()
+  }, [me, navigate, startPollPlayers])
+
+  return (
+    <div>
+      <div className="side-img">
+        <img src={Oberin} alt="Oberin" />
+      </div>
+      <div className="box">
+        <div className="box-left">
+          <img className="image" src={Title} alt="title" />
+        </div>
+        <div className="box-right">
+          <form onSubmit={submit}>
+            <p className="title">LOGIN</p>
```

Comment:
```suggestion
            <p className="title">create player</p>
```
Ich würde hier das all caps mit 
text-tranform: uppercase;
im css machen, damit bei änderung des cases der test nicht auseinander fällt.

Reply:
Der Test ist meines Erachtens überflüssig...

Reply:
Nein, ist er sicher nicht. (und da machst du mich hässig).
Es ist ein einfacher test, der mal schaut ob das ganze wiring funktioniert. Mit bis jetzt wenig aufwand hilft er viel.

Reply:
Alles klar. Habe ich somit bisher nicht verstanden.